src/core/messages.ts

```diff
@@ -479,7 +499,7 @@ export class DefaultMessages
    */
   subscribe(listener: MessageListener): MessageSubscriptionResponse {
```

Comment:
An API question here - should we be allowing users to provide events that they want to subscribe to, a-la presence?

Reply:
> Ah yeh, I think we should, I'll make the change to support this as part of this PR.

Doesn't have to be as part of this PR, but would be good to get a DR around it nonetheless - I think there's a discussion to be had around whether we do optional first args (I really dislike the optional first arg pattern!) or whether we do an options object.